build/charts/theia/values.yaml

```diff
@@ -150,7 +146,40 @@ clickhouse:
       # Use a specific uid, gid for zookeeper.
       securityContext:
         runAsUser: 1000
-        fsGroup: 1000 
+        fsGroup: 1000
+      storage:
+        # -- Zookeeper storage size. Can be a plain integer or as a fixed-point
+        # number using one of these quantity suffixes: E, P, T, G, M, K. Or the
+        # power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki.
+        size: "5Gi"
+        createPersistentVolume:
```

Comment:
Do you think we need to also enable ZooKeeper PV in the e2e when we enable PV for ClickHouse? (I think only the migrate_clickhouse_test is using ClickHouse with PV for now. Please tell me if I miss any other test.)

Reply:
We currently use ClickHouse with PV for `migrate_clickhouse_test` and `upgrade_test`.
I think it's safer to enable ZooKeeper PV when enabling PV for ClickHouse.
Do you think we need to use a separate version? Like only enable ZooKeeper for version higher than v0.8.0? Or I can just enable it whenever --local flag is used?

Reply:
I also found an issue that when upgrading the ZooKeeper from default one to the one with PV, the data will be completely lost, leading ClickHouse to keep crashing again.

Reply:
We discussed offline and agreed that we could enable ZooKeeper PV in e2e tests in next release to avoid the upgrading issue between in-memory and PV